src/webui/components/Icon/icons/License.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import SvgIcon, { SvgIconProps } from '../../SvgIcon'
+
+const License: React.FC<SvgIconProps> = props => (
+  <SvgIcon viewBox="0 0 490.652 490.652" {...props}>
```

Comment:
Remove original `svg` files is not good idea. Reasons:

# This blow up the JS bundle 

### Master branch

```
  0.style.cdd1fdf79cd6bec30c42.css   1.91 KiB       0  [emitted]         vendors
     vendors.bb36852670bb2ecbf5d9.js    593 KiB       0  [emitted]  [big]  vendors
           1.bb36852670bb2ecbf5d9.js   20.7 KiB       1  [emitted]         
        main.bb36852670bb2ecbf5d9.js    123 KiB       2  [emitted]         main
           3.bb36852670bb2ecbf5d9.js   6.88 KiB       3  [emitted]         
           4.bb36852670bb2ecbf5d9.js    118 KiB       4  [emitted]         
           5.bb36852670bb2ecbf5d9.js   25.6 KiB       5  [emitted]         
    6.style.ad898c44026f788f7d59.css   11.1 KiB       6  [emitted]         
           6.bb36852670bb2ecbf5d9.js   31.1 KiB       6  [emitted]         
    manifest.bb36852670bb2ecbf5d9.js   3.25 KiB       7  [emitted]         manifest
                         favicon.ico   14.7 KiB          [emitted]         
                          index.html   1.29 KiB          [emitted]         
```
vs this branch
```
  0.style.cdd1fdf79cd6bec30c42.css   1.91 KiB       0  [emitted]         vendors
     vendors.46620792069afcb3c751.js    633 KiB       0  [emitted]  [big]  vendors
           1.46620792069afcb3c751.js   20.7 KiB       1  [emitted]         
        main.46620792069afcb3c751.js    146 KiB       2  [emitted]         main
           3.46620792069afcb3c751.js   6.96 KiB       3  [emitted]         
           4.46620792069afcb3c751.js    119 KiB       4  [emitted]         
           5.46620792069afcb3c751.js   21.5 KiB       5  [emitted]         
    6.style.ad898c44026f788f7d59.css   11.1 KiB       6  [emitted]         
           6.46620792069afcb3c751.js     31 KiB       6  [emitted]         
    manifest.46620792069afcb3c751.js   3.25 KiB       7  [emitted]         manifest
                         favicon.ico   14.7 KiB          [emitted]         
                          index.html   1.29 KiB          [emitted]         
```

Reply:
`@material-ui/styles` still alpha and unstable according to their docs. 

https://material-ui.com/css-in-js/basics/
https://github.com/mui-org/material-ui/issues/13663
https://github.com/mui-org/material-ui/milestones